provider/workspace_owner.go

```diff
@@ -59,6 +59,14 @@ func workspaceOwnerDataSource() *schema.Resource {
 				_ = rd.Set("login_type", loginType)
 			}
 
+			var rbacRoles []string
+			if rolesRaw, ok := os.LookupEnv("CODER_WORKSPACE_OWNER_RBAC_ROLES"); ok {
+				if err := json.NewDecoder(strings.NewReader(rolesRaw)).Decode(&rbacRoles); err != nil {
+					return diag.Errorf("invalid user rbac roles: %s", err.Error())
+				}
+			}
+			_ = rd.Set("rbac_roles", rbacRoles)
```

Comment:
`CODER_WORKSPACE_OWNER_RBAC_ROLES` is now a data structure right?

https://github.com/coder/coder/blob/77479cdd51f5154054e27734b30900a01f014729/provisionersdk/proto/provisioner.proto#L258-L261

So we need to json decode the env var, and correct the schema to be an object with 2 fields: `name` and `org_id`.

Reply:
Correct - I plan to revisit this today or tomorrow and make the updates based on the changes in the other PR

Reply:
@nxf5025 awesome, just keep pinging me :+1:

Reply:
@Emyrk - Just pushed an update. Let me know if any changes are needed